web3.js/test/connection.test.js

```diff
@@ -1521,6 +1521,27 @@ test('get largest accounts', async () => {
   expect(largestAccounts.length).toEqual(20);
 });
 
+test('get stake activation', async () => {
```

Comment:
A few passing tests would be nice here too!

Reply:
added an extra one to check the state literals